### DISCUSSION TITLE
How to make a some style image?

### DISCUSSION THREAD

**Discussion by K-O-N-B** [cubiq/ComfyUI_IPAdapter_plus#384] [Q&A]
here is the ref image
![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/152141077/779cf648-bcec-4e5d-bc29-6f150ac6f579)

I love the night and rainning scenes, so I try to copy the style and generate some new images. 

![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/152141077/d308ed16-5c74-4d87-bc74-721d1ea3476c)
![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/152141077/3c1a36d2-3f98-41b7-af87-bf47e5a34090)

but I've try many IPadapter's options(weight_type/embeds_scaling/weight) and prompts and checkpoint,  
the result is very bad..... cry

**Comment by K-O-N-B** [comment]
oh, I try to use SD15, the result is better now
![image](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/152141077/1a1195d3-c356-43e5-8c93-1196814f249e)

it looks not really good work at sdxl.

**Comment by cubiq** [comment]
seems to be working well with SDXL too, try different checkpoints

![ComfyUI_temp_qazaq_00124_](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/b552f780-c160-4c31-b8f6-3b0c75acc9ca)

**Reply by K-O-N-B** [reply]
wow, what checkpoint is it? and could you share the workflow pls?

**Reply by K-O-N-B** [reply]
that's amazing!

**Reply by cubiq** [reply]
turbovision
